### ISSUE TITLE
Group Questing and Looting

### ISSUE THREAD

**Issue by thestranger064** [dalaranwow/dalaran-wow#5424]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 
When you form a group and do quests only 1 person can loot said objects or npc.**: When you kill an enemy only 1 person can loot the quest item, but the enemy still glitters and wont allow other players to click it.

When you loot an object on ground only 1 person can take it, not the entire group, that should have a tag for it.

**Expected behaviour**: When forming a group for quest objectives, it should allow things to be easier, not harder and more frustrating.
Entire group should be able to loot the current object, or make it instant respawn.
Entire group should be able to loot said npc on ground once killed, and not just one player.

**Steps to reproduce the problem**:

1. Form a group, get a quest in the world somewhere.
2.  Loot a corpse or take a object from the ground.
3. Observe and see that the rest of the group has to wait for new respawns. 

**Include proofs for this behaviour**
I have done many quests since my return to dalaran and I am in Northrend right now, and every quest is the same, as well as quest objects on the ground. 
This is very frustrating when you play in groups, but solo players will not notice this.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

I dont really see why you need specifics, since it involves almost every quest, object in the world, but here is a few.

https://wotlk.evowow.com/?quest=13285 -  Forging the keystone (have to wait more then 30m for a respawn if another player in group has done it, and not every player get credit.)

https://wotlk.evowow.com/?quest=12858 - Pieces of the puzzle (Another mob drop thing only 1 person can click and get.)

https://wotlk.evowow.com/?quest=12860 - Data Mining (Quest objects only 1 person can tag, get credit for even in party or group.)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
You don't provide any proof for the behavior. What you just reported are the intended mechanics of group questing back in WotLK, with its advantages and disadvantages. [_Proof example_](https://us.battle.net/forums/en/wow/topic/4427533286)

Please take this one to the [Suggestions and Opinions](http://www.dalaran-wow.com/forums/news-and-rules/suggestions-and-opinions) section of the forums.

Closing

**Comment by thestranger064** [comment]
I listed quests as examples  that didnt work in a group, and you linked to a opinion thread as a proof.
You closed a thread that is a mayor issue, because you think your opinion as proof is more valid.

**Comment by Golganeth** [comment]
Not an opinion, this was the intended behavior back in WotLK and older expansions. If you wish to report a quest that doesn't work as intended, or a game object that takes too long to respawn so that other party members can loot it faster, please **make an specific report since most of them are handled individually, this will make developers work a little bit easier.**

#5428
